py/head.py

```diff
@@ -2,18 +2,56 @@
 from dominate.tags import *
 
 def _head():
-    """Returns the generated HTML header. TODO: edit this description
-
-    TODO: A description here, according to Google's Python style guide https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings
+    """This function generates a header using table elements and returns it as HTML. The header consists of the banner logo
+    (lightbulbs and title) and the social media handles with their icons. The banner links to the main TSE website and each
+    social media handle links to their respective accounts. This section of the newsletter should not need to be changed
+    since no new information needs to be added. DO NOT EDIT!
 
     Args:
-        TODO
+        None
 
     Returns:
-        TODO
+        The generated HTML header. The header consists of table elements split into two rows. The top row contains the banner and the
+        bottom row contains the social media handles. Within each row are more tables and columns containing the contents with anchor tags, 
+        namely the lightbulbs, title, social media icons, and social media handles.
 
     Raises:
-        TODO
+        None
     """
+    with doc:
```

Comment:
I believe this wraps the entire things in a `<body>` tag, which is not what we want. What is the output we get when you run `python newsletter.py`?

Reply:
Good point. I forgot to change it after modeling head.py after header.html